charts/values.yaml

```diff
@@ -0,0 +1,415 @@
+    name: openebs-hostpath
+    # If true, enables creation of the openebs-hostpath StorageClass
+    enabled: true
+    # Available reclaim policies: Delete/Retain, defaults: Delete.
+    reclaimPolicy: Delete
+    # If true, sets the openebs-hostpath StorageClass as the default StorageClass
+    isDefaultClass: false
+    # Path on the host where local volumes of this storage class are mounted under.
+    # NOTE: If not specified, this defaults to the value of localpv.basePath.
+    basePath: ""
+    # Custom node affinity label(s) for example "openebs.io/node-affinity-value"
+    # that will be used instead of hostnames
+    # This helps in cases where the hostname changes when the node is removed and
+    # added back with the disks still intact.
+    # Example:
+    #          nodeAffinityLabels:
+    #            - "openebs.io/node-affinity-key-1"
+    #            - "openebs.io/node-affinity-key-2"
+    nodeAffinityLabels: []
+    # Prerequisite: XFS Quota requires an XFS filesystem mounted with
+    # the 'pquota' or 'prjquota' mount option.
+    xfsQuota:
+      # If true, enables XFS project quota
+      enabled: false
+      # Detailed configuration options for XFS project quota.
+      # If XFS Quota is enabled with the default values, the usage limit
+      # is set at the storage capacity specified in the PVC.
+      softLimitGrace: "0%"
+      hardLimitGrace: "0%"
+    # Prerequisite: EXT4 Quota requires an EXT4 filesystem mounted with
+    # the 'prjquota' mount option.
+    ext4Quota:
+      # If true, enables XFS project quota
+      enabled: false
+      # Detailed configuration options for EXT4 project quota.
+      # If EXT4 Quota is enabled with the default values, the usage limit
+      # is set at the storage capacity specified in the PVC.
+      softLimitGrace: "0%"
+      hardLimitGrace: "0%"
+
+  helperPod:
+    image:
+      registry: ""
+      repository: openebs/linux-utils
+      pullPolicy: IfNotPresent
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: 4.0.0
+
+  analytics:
+    enabled: true
+    # Specify in hours the duration after which a ping event needs to be sent.
+    pingInterval: "24h"
+
+mayastor:
+  # -- Enable Mayastor storage engine
+  enabled: true
+  crds:
+    enabled: false
+  # Sample configuration, if you want to configure mayastor with custom values.
+  # This is a small part of the full configuration. Full configuration available
+  # here - https://github.com/openebs/mayastor-extensions/blob/v2.5.0/chart/values.yaml
+
+  image:
+    # -- Image registry to pull Mayastor product images
+    registry: docker.io
+    # -- Image registry's namespace
+    repo: openebs
+    # -- Release tag for Mayastor images
+    tag: v2.6.0
+    # -- ImagePullPolicy for Mayastor images
+    pullPolicy: IfNotPresent
+
+  # -- Pod scheduling priority
+  # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  # priorityClassName: ""
+
+  # base:
```

Comment:
Do we need all these commented out values? Some of them are probably outdated now which would confuse users...

Reply:
We can remove them probably. And a link to individual chart docs?

Reply:
@niladrih @avishnu Does that sound good?